client/src/reducers/index.js

```diff
@@ -124,4 +126,13 @@ export default combineReducers({
                 return conversation;
         }
     },
+    switchView: (initialSwitchState = switchState, action) => {
```

Comment:
**NITPICKING**, but could call `initialSwitchState` -> `switchState`, and `switchState` -> `defaultSwitchState` to abide with the general convention.

Reply:
Done!